### PR TITLE
Chore: update issue template with --eslint-fix flag

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain: 
updated the issue template accordingly such that it instructs the user to use `eslint --env-info` to find information converning his/her local environment.

**What changes did you make? (Give an overview)**
`v6.5.0` release introduced a new `--env-info` flag which would show up the local environment information. Updated the issue-template accordingly.

**Is there anything you'd like reviewers to focus on?**
Nope

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -13,9 +13,7 @@
 
 **Tell us about your environment**
 
-* **ESLint Version:**
-* **Node Version:**
-* **npm Version:**
+paste the results of **eslint --env-info** here.
```
Comment (kaicataldo) [inline .github/ISSUE_TEMPLATE.md]: I think we should leave the current text and put a note in a markdown comment that lets people know about the new flag. A lot of folks are still going to be on older versions of ESLint that don't have this flag.

```md
<!--
    If you are using ESLint v6.5.0 or later, you can run ESLint with the `--env-info` flag and paste the output here.
-->

* **ESLint Version:**
* **Node Version:**
* **npm Version:**
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -22,9 +22,7 @@ assignees: ''
 
 **Tell us about your environment**
 
-* **ESLint Version:**
-* **Node Version:**
-* **npm Version:**
+paste the results of **eslint --env-info** here.
```
Comment (kaicataldo) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: Same comment.

Comment (jamesgeorge007) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md]: :+1:

**Comment by jamesgeorge007** [comment]
\cc @kaicataldo

**Comment by kaicataldo** [comment]
Thanks for contributing!

**Pull request merged**